Novos Negocios

ToxiK0 opened the pull request:
Criei os novos negocios, falta fazer o toString das novas subclasses

ToxiK0 committed:
Novos Negocios

Criei os novos negocios, falta fazer o toString das novas subclasses

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Armas.java:
```diff
@@ -0,0 +1,39 @@
+    }
+    
+    //Média do Musculo da equipa de Soldiers
+    public void VerificaMusculo(){
```
@ToxiK0 isto nao era para ficar dentro da classe negocio?

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/CasinoIlegal.java:
```diff
@@ -0,0 +1,43 @@
+package com.mycompany.projeto2;
+
+public class CasinoIlegal extends Negocio{
```
@ToxiK0 acho que o nome pode ficar apenas "Casino"

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/CasinoIlegal.java:
```diff
@@ -0,0 +1,43 @@
+    }
+    
+    //Média do Musculo da equipa de Soldiers
+    public void VerificaInteligencia(){
```
@ToxiK0 isto nao era para ficar dentro da classe negocio?

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Roubo.java:
```diff
@@ -0,0 +1,43 @@
+    }
+    
+    //Média do Musculo da equipa de Soldiers
+    public void VerificaMusculoEInteligencia(){
```
@ToxiK0 isto nao era para ficar dentro da classe negocio?

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/LavagemDinheiro.java:
```diff
@@ -0,0 +1,39 @@
+    }
+    
+    //Média do Musculo da equipa de Soldiers
+    public void VerificaInteligencia(){
```
@ToxiK0 isto nao era para ficar dentro da classe negocio?

ToxiK0 committed:
Atualizei isto tudo

Mudei o nome do casino ilegal para casino e passei o metodo verificamusculo e verificainteligencia para a classe negocio.

P3dr0jardim commented on Projeto2/src/main/java/com/mycompany/projeto2/Armas.java:
```diff
@@ -0,0 +1,31 @@
+package com.mycompany.projeto2;
+
+public class Armas extends Negocio{
+    private double rentabilidade;
```
Nao tenho a certeza mas acho que isto podia estar na classe negocio, que dizes?

P3dr0jardim committed:
Rentabilidade improvement
